.gitignore

```diff
@@ -66,3 +66,7 @@ bin
 op-defender/.air.toml
 op-defender/run.sh
 op-defender/tmp
+
+local
+
+.venv
```

Comment:
I don't think we have python venv maybe forgotten?

Reply:
I have local python and will also add later.